src/pages/mission/entry/MissionEntry.tsx

```
@@ -80,25 +81,67 @@ function MissionEntryContent({ missionId }: { missionId: number }) {
   };
 
   useEffect(() => {
-    const handleVisibilityChange = () => {
-      if (
-        document.visibilityState === "visible" &&
-        clickedAtRef.current &&
-        !isContentWatched
-      ) {
-        const elapsed = Date.now() - clickedAtRef.current;
+    if (!isContentWatched) {
+      const savedClickedAt = sessionStorage.getItem("missionClickedAt");
```

Biome (2.3.14)

[error] This hook does not specify its dependency on callWatchApi.

This dependency is being used here, but is not specified in the hook dependency list.

This dependency is being used here, but is not specified in the hook dependency list.

React relies on hook dependencies to determine when to re-compute Effects.
Failing to specify dependencies can result in Effects not updating correctly when state changes.
These "stale closures" are a common source of surprising bugs.
Either include it or remove the dependency array.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)